### ISSUE TITLE
[go] disneynow.go.com; DirectTV unable to extract POST url

### ISSUE THREAD

**Issue by shirtjs** [rg3/youtube-dl#18901]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2019.01.17*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2019.01.17**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
[debug] System config: []
[debug] User config: ['--ap-mso', 'DTV', '--ap-username', 'PRIVATE', '--ap-password', 'PRIVATE']
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://disneynow.go.com/shows/pokemon-the-series-sun-moon2/season-21/episode-42-i-choose-paradise/vdka7464081?pid=PL553044961']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2019.01.17
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17763
[debug] exe versions: ffmpeg N-92932-g038d291b70, ffprobe N-92932-g038d291b70
[debug] Proxy map: {}
[Go] vdka7464081: Downloading webpage
[Go] VDKA7464081: Downloading JSON metadata
[Go] VDKA7464081: Downloading Provider Redirect Page
ERROR: Unable to extract post url; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpgf__qgjd\build\youtube_dl\YoutubeDL.py", line 793, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpgf__qgjd\build\youtube_dl\extractor\common.py", line 508, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpgf__qgjd\build\youtube_dl\extractor\go.py", line 137, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpgf__qgjd\build\youtube_dl\extractor\adobepass.py", line 1505, in _extract_mvpd_auth
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpgf__qgjd\build\youtube_dl\extractor\adobepass.py", line 1360, in post_form
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpgf__qgjd\build\youtube_dl\extractor\common.py", line 992, in _html_search_regex
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpgf__qgjd\build\youtube_dl\extractor\common.py", line 983, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract post url; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
...
<end of log>
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://disneynow.go.com/shows/pokemon-the-series-sun-moon2/season-21/episode-42-i-choose-paradise/vdka7464081

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information

https://disneynow.go.com/ videos fail to get to adobepass and it gives the error "Unable to extract post url".

**Comment by shirtjs** [comment]
Can you explain why this was marked as a duplicate? All the other issues were for Comcast, while this is DirecTV.

**Comment by remitamine** [comment]
Duplicate of #12997.

**Comment by remitamine** [comment]
@shirtjs I was wrong about your issue, the error was similar to #12997 but the cause was different(specific to Disney Now), I have pushed a fix for the Adobe Pass requests in 118afcf52ff23726e5f0c436083710f5c63230fa.
